### PR TITLE
Step13state

### PR DESCRIPTION
**Pull request by S-B1**
Attempting to create a second pull request. 
Thank you for selecting to watch Ricardo.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,14 +1,27 @@
+/* Repaired text colour issue using a tag */
+
 .App {
   text-align: center;
-  color: rgb(180, 110, 4);
+  color: black;
+}
+/* changed the text colour -> blk header */
+
+
```
Comment (rmallols) [inline src/App.css]: Please remove unnecessary space

File: src/App.css [hunk 2]
```diff
@@ -1,14 +1,27 @@
+/* Repaired text colour issue using a tag */
+
 .App {
   text-align: center;
-  color: rgb(180, 110, 4);
+  color: black;
+}
+/* changed the text colour -> blk header */
```
Comment (rmallols) [inline src/App.css]: Please remove comments, if you want to explain your changes, please drop a message on the PR

File: src/App.css [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
+
+
+a {
+  color: rgb(177, 109, 6);
 }
+/* added a tag to counter the effect of the colour change */
 
 .container {
-  display: flex;
+  /* display: flex;
```
Comment (rmallols) [inline src/App.css]: Please remove commented code - old stuff can be reviewed on the Git history

File: src/Stack.js [hunk 4]
```diff
@@ -1,15 +1,18 @@
...
+// #11 Improve Gallery Layout for lge screen
+
 
 import React from 'react';
 import {Link} from 'react-router-dom';
 
 export default function Stack(props) {
     return(
-        <Link to='/details' className='item'>
+        <Link to={'/' + props.id} className='item'>
```
Comment (rmallols) [inline src/Stack.js]: Could you please use back-ticks here?

File: src/App.css (hunk 1 above)
Comment (S-B1) [reply src/App.css]: Done. Thank you

File: src/App.css (hunk 2 above)
Comment (S-B1) [reply src/App.css]: Done. Thank you

File: src/App.css (hunk 3 above)
Comment (S-B1) [reply src/App.css]: Done. Thank you

File: src/Stack.js (hunk 4 above)
Comment (S-B1) [reply src/Stack.js]: Is this what you are referring to?
<Link to={`/${'/' + props.id}`} className='item'>

**Pull request merged**